client/src/Routes.jsx

```diff
@@ -5,6 +5,7 @@ import {
 
 import Login from './views/Login';
 import Signup from './views/Signup';
+import New from './views/New';
```

Comment:
can we name it better ?

Reply:
Here I'm following the convention of naming components after the route they render in, which in this case is `/new`. But agree that it's not a good component name.
`NewPost` was already taken.